sfge/src/test/java/com/salesforce/rules/fls/apex/ValidationConverterTest.java

```diff
@@ -284,7 +284,10 @@ public boolean visit(DmlStatementVertex vertex, SymbolProvider symbols) {
                                 new ValidationConverter(FlsValidationType.UPSERT);
                         validationReps.addAll(
                                 converter.convertToExpectedValidations(
-                                        ScopeUtil.resolveToApexValue(symbols, childVertex).get()));
+                                        ScopeUtil.resolveToApexValue(symbols, childVertex).get(),
+                                        // false because DmlStatements don't have a way to set
+                                        // access level
+                                        false));
```

Comment:
Technically, Dml statements _do_ have a way to set access level. You can do `as user` or `as system` in your transaction to set that.
E.g., `insert as user someAccount`.
However, this is handled elsewhere, so you're correct that it doesn't need to be handled here.

Reply:
Removed due to refactor.